caldp/messages.py

```diff
@@ -186,16 +186,16 @@ def upload_message(self):
     def sync_dataset(self):
         if self.output_uri.startswith("file"):
             preview_output = os.path.join(self.output_path, "previews")
-            files = glob.glob(f"{self.output_path}/{self.ipppssoot[0:5]}*")
-            files.extend(glob.glob(f"{preview_output}/{self.ipppssoot[0:5]}*"))
+            files = glob.glob(f"{self.output_path}/{self.dataset[0:5]}*")
```

Comment:
Will this code might break due to the changes in syntax of ipppssoot vs. dataset?